src/lib/arch/aarch64/ExceptionHandler.cc

```diff
@@ -586,6 +592,37 @@ bool ExceptionHandler::init() {
         stateChange = {ChangeType::REPLACEMENT, {R0}, {0ull}};
         break;
       }
+      case 278: {  // getrandom
+        // TODO: support flags argument
```

Comment:
Ideally the flags should be supported before this PR is merged

Reply:
From the man pages, the flags are:
```
GRND_RANDOM
              If this bit is set, then random bytes are drawn from the
              random source (i.e., the same source as the /dev/random
              device) instead of the urandom source.  The random source
              is limited based on the entropy that can be obtained from
              environmental noise.  If the number of available bytes in
              the random source is less than requested in buflen, the
              call returns just the available random bytes.  If no
              random bytes are available, the behavior depends on the
              presence of GRND_NONBLOCK in the flags argument.

       GRND_NONBLOCK
              By default, when reading from the random source,
              getrandom() blocks if no random bytes are available, and
              when reading from the urandom source, it blocks if the
              entropy pool has not yet been initialized.  If the
              GRND_NONBLOCK flag is set, then getrandom() does not block
              in these cases, but instead immediately returns -1 with
              [errno](https://man7.org/linux/man-pages/man3/errno.3.html) set to EAGAIN.
```
Do we want to simulate this blocking or the EAGAIN signal? If not then perhaps it is safe to ignore the flags after all?